security/userprovided-tls_mtls_kafka-acls/README.md

```diff
@@ -0,0 +1,372 @@
+# Security setup
+
+In this workflow scenario, you`ll set up a Confluent Platform cluster with the following security:
+- Full TLS network encryption with user provided certificates
+- mTLS authentication
+- Kafka ACLs authorization
+
+Before continuing with the scenario, ensure that you have set up the [prerequisites](https://github.com/confluentinc/confluent-kubernetes-examples/blob/master/README.md#prerequisites).
+
+## Set the current tutorial directory
+
+Set the tutorial directory for this tutorial under the directory you downloaded the tutorial files:
+
+```
+export TUTORIAL_HOME=<Tutorial directory>/security/userprovided-tls_mtls_kafka-acls
+```
+  
+## Deploy Confluent for Kubernetes
+
+Set up the Helm Chart:
+
+```
+helm repo add confluentinc https://packages.confluent.io/helm
+```
+
+Install Confluent For Kubernetes using Helm:
+
+```
+helm upgrade --install operator confluentinc/confluent-for-kubernetes --namespace confluent
+```
+  
+Check that the Confluent For Kubernetes pod comes up and is running:
+
+```
+kubectl get pods --namespace confluent
+```
+
+## Create TLS certificates
+
+In this scenario, you`ll configure authentication using the mTLS mechanism. With mTLS, Confluent components and clients use TLS certificates for authentication. The certificate has a CN that identifies the principal name.
+
+Each Confluent component service should have it`s own TLS certificate. In this scenario, you`ll
+generate a server certificate for each Confluent component service. Follow [these instructions](../../assets/certs/component-certs/README.md) to generate these certificates.
+
+These TLS certificates include the following principal names for each component in the certificate Common Name:
+- Kafka: `kafka`
+- Schema Registry: `sr`
+- Kafka Connect: `connect`
+- ksqlDB: `ksql`
+- Control Center: `controlcenter`
+     
+## Deploy configuration secrets
+
+You`ll use Kubernetes secrets to provide credential configurations.
+
+With Kubernetes secrets, credential management (defining, configuring, updating)
+can be done outside of the Confluent For Kubernetes. You define the configuration
+secret, and then tell Confluent For Kubernetes where to find the configuration.
+
+To support the above deployment scenario, you need to provide the following
+credentials:
+
+* Component TLS Certificates
+
+* Authentication credentials for Zookeeper, Kafka, Control Center, remaining CP components
+
+### Provide component TLS certificates
+
+Set the tutorial directory for this tutorial under the directory you downloaded the tutorial files:
+
+```
+export TUTORIAL_HOME=<Tutorial directory>/security/userprovided-tls_mtls_confluent_rbac
```

Comment:
should this be `export TUTORIAL_HOME=<Tutorial directory>/security/userprovided-tls_mtls_kafka-acls` ?